apps/meteor/ee/app/livechat-enterprise/server/api/lib/priorities.ts

```diff
@@ -37,11 +37,19 @@ export async function findPriority({
 	};
 }
 
-export async function updatePriority(_id: string, data: Pick<ILivechatPriority, 'name'>): Promise<ILivechatPriority> {
+export async function updatePriority(
+	_id: string,
+	data: Pick<ILivechatPriority & { reset: boolean }, 'name' | 'reset'>,
+): Promise<ILivechatPriority> {
 	const query = {
 		_id,
 	};
-
-	const created = await LivechatPriority.findOneAndUpdate(query, { $set: { ...data, dirty: true } }, { returnDocument: 'after' });
+	const update: { $set: Record<string, boolean>; $unset?: Record<string, 1 | ''> } = { $set: { dirty: !data.reset } };
```

Comment:
Can you use `Pick<UpdateOperation<ILivechatPriority>, '$set'>` 👀

Reply:
UpdateOperation does not exist here, but UpdateFilter works.